apps/builder/app/shared/nano-states/misc.ts

```diff
@@ -324,6 +335,54 @@ export const $authTokenPermissions = atom<TokenPermissions>({
 
 export const $authToken = atom<string | undefined>(undefined);
 
+export const $isContentModeAllowed = computed(
+  [$authToken, $userPlanFeatures],
+  (token, userPlanFeatures) => {
+    return (
+      token === undefined ||
+      (token !== undefined && userPlanFeatures.hasProPlan === true)
```

Comment:
this
(
      token === undefined ||
      (token !== undefined && userPlanFeatures.hasProPlan === true)

 is basically the same as
 
(
      token === undefined ||
     userPlanFeatures.hasProPlan === true